Dokumentationen/test/test_cases.adoc

```diff
@@ -512,12 +495,13 @@ a| *
 
 |===
 
-|*Beschreibung* |
+|*Beschreibung* | Das System wird in alles geforderten Browser auf volle Funktionalität getestet. Es wird erwartet, dass das System in allen getesteten Browsern vollumfänglich und mit gleichen Ergebnissen funktioniert.
 
-|*Vorbedingungen und benötigte Daten*
-a| * 
+|*Vorbedingungen und benötigte Daten und benötigte Daten*
+a| * ein Admin-, ein Dozenten- und ein Gastkonto
+* verschiedene Browser
 
 |*Nachbedingung*
-a| *
+a| * alle Funktionen des Systems funktionieren in den unterschiedlichen Browsern wie erwartet und entsprechend den Berechtigungen der einzelnen Accounts
```

Comment:
Ich würde bei dem TC eig gern hinzufügen, dass auch die Darstellung korrekt ist (dies ist oft auch ein Problem in verschiedenen Browsern). Sollen wir das als extra Punkt zu den Nachbedingungen hinzufügen oder seht ihr das bei den Funktionen mit inbegriffen?

Reply:
Ich habe noch einen extra Punkt hinzugefügt